exercises/octal/description.md

```diff
@@ -1,4 +1,6 @@
-Convert a octal number, represented as a string (e.g. '1735263'), to its decimal equivalent using first principles (i.e. no, you may not use built-in or external libraries to accomplish the conversion).
+Convert a octal number, represented as a string (e.g. '1735263'), to its
```

Comment:
> Convert a octal number

should probably read

> Convert an octal number

(This exercise is deprecated but likely still served by some tracks, so I figure the change is worth making?)